src/skmultiflow/lazy/knn_classifier.py

```diff
@@ -164,49 +158,26 @@ def predict_proba(self, X):
         Returns
         -------
         numpy.ndarray
-            An array of shape (n_samples, n_features), in which each outer entry is 
-            associated with the X entry of the same index. And where the list in 
-            index [i] contains len(self.target_value) elements, each of which represents
-            the probability that the i-th sample of X belongs to a certain label.
+            A  2D array of shape (n_samples, n_classes). Where each i-row
```

Comment:
What about "each i-th row"

Reply:
Updated in 31687ca4743f3fa03f752ed182135cbef9fc7901